```
[2022-01-21T11:46:12.299Z] net.corda.node.NodeRPCTests > run nodeDiagnosticInfo FAILED
[2022-01-21T11:46:12.299Z]     java.lang.AssertionError: Expected <11>, actual <10>.
[2022-01-21T11:46:12.299Z]         at kotlin.test.DefaultAsserter.fail(DefaultAsserter.kt:17)
```

--- a/constants.properties
+++ b/constants.properties
@@ -11,7 +11,7 @@ java8MinUpdateVersion=171
 # When incrementing platformVersion make sure to update          #
 # net.corda.core.internal.CordaUtilsKt.PLATFORM_VERSION as well. #
 # ***************************************************************#
-platformVersion=10
+platformVersion=11
 guavaVersion=28.0-jre
 # Quasar version to use with Java 8:
 quasarVersion=0.7.13_r3

--- a/core/src/main/kotlin/net/corda/core/internal/CordaUtils.kt
+++ b/core/src/main/kotlin/net/corda/core/internal/CordaUtils.kt
@@ -28,6 +28,8 @@ import java.util.jar.JarInputStream
 
 // *Internal* Corda-specific utilities.
 
+
+// When incrementing platformVersion make sure to update PLATFORM_VERSION in constants.properties as well.
 const val PLATFORM_VERSION = 11
 
 fun ServicesForResolution.ensureMinimumPlatformVersion(requiredMinPlatformVersion: Int, feature: String) {

